lib/karafka/controllers/includer.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Karafka
+  # Additional functionalities for controllers
+  module Controllers
+    # Module used to inject functionalities into a given controller class, based on the controller
+    # topic and its settings
+    # We don't need all the behaviors in all the cases, so it is totally not worth having
+    # everything in all the cases all the time
+    module Includer
+      # @param controller_class [Class] controller class, that will get some functionalities
+      #   based on the topic under which it operates
+      def self.call(controller_class)
```

Comment:
This can also be a great way for other libraries to build prefab stuff into controllers.

Reply:
Yeah, we can come up with something more flexible in the future, but I agree, that this is a good start